### PR TITLE
Particle level hjet histograms were added

### PR DESCRIPTION
**Pull request by fkrizek**

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.cxx [hunk 1]
```diff
@@ -1020,20 +1151,68 @@ Bool_t AliAnalysisTaskEA::FillHistograms(){
...
+
+
    if(fMC){
+      TClonesArray* arrayMC = 0; // array particles in the MC event
+      arrayMC = (TClonesArray*) InputEvent()->FindListObject(AliAODMCParticle::StdBranchName());
+      if(!arrayMC){
+         AliError("No MC array found!");
+         return kFALSE;
+      }
+      AliAODMCParticle* daughtermc;
```
Comment (mfasDa) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.cxx]: I recommend to use the AliParticleContainer, in particular for the Photon double counting bug you cannot rely that the particles are in the standard branch.

**Comment by mfasDa** [comment]
+test

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.cxx (hunk 1 above)
Comment (fkrizek) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.cxx]: Hi Markus,
in the standard branch i am looking just for the daughters of the photon. If the daughter is another photon, i discard such particle.     Or should i apply some standard approach how to filter them out?
In such a case could you point me to the piece of code where this is handeled

Many thanks
Filip

Comment (mfasDa) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.cxx]: Hi @fkrizek ,

The code is OK, but in principle we have a preparation task doing this called AliEmcalMCTrackSelector (https://github.com/alisw/AliPhysics/blob/master/PWG/EMCAL/EMCALtasks/AliEmcalMCTrackSelector.h). This task however doesn't necessarily write into the standard branch. The result from the track selection is interfaced via the AliMCParticleContainer. I recommend using standard tools.

In the mean time I will approve the PR.

Cheers

Markus

**Comment by mfasDa** [comment]
+1

**Pull request merged**